server/game/cards/12-KotI/DesertRaider.js

```diff
@@ -0,0 +1,52 @@
+const DrawCard = require('../../drawcard.js');
+const ChallengeTypes = require('../../ChallengeTypes');
+
+class DesertRaider extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                afterChallenge: event => event.challenge.loser === this.controller && this.controller.canPutIntoPlay(this)
+            },
+            location: 'dead pile',
+            target: {
+                type: 'select',
+                cardCondition: card => card.location === 'play area' && card.getType() === 'character' &&
+                                       card.isParticipating() && card.canGainPower() && this.game.currentChallenge.winner === card.controller
```

Comment:
Card text doesn't require the character to be participating in the challenge, just controlled by the winning opponent. It probably also doesn't need to be able to actually gain the power - it reads like a dual effect - gains 1 power AND loses an icon. As long as one can happen, you should be able to target the character.